tutorial/creating_packages/other_packages/toolchain_packages/toolchain/conanfile.py

```diff
@@ -0,0 +1,80 @@
+import os
+from conan import ConanFile
+from conan.tools.files import get, copy, download
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+
+# conan create toolchain.py -pr:b=default -pr:h=./profiles/raspberry-64 --build-require -c:b tools.files.download:verify=False
```

Comment:
Is ``tools.files.download:verify=False`` necessary? Only for us?

Reply:
Sorry, that was I left that as a comment for myself and forgot to remove it. Removing it.